source/getting-started/configuration.rst

```diff
@@ -1,9 +1,119 @@
 Configuration
 -------------
 
-Minimal "sane" config:
+The configuration of Unbound can be a little tricky due to the extensive array of configurable options. Below we will go through a basic, recommended config, but feel free to add and experiment with options as you need them. Also feel free to remove lines that are commented out (uncommenting bu removing the "#") if they are not nesecary in your setup.
+
+For the configuration step, we will assume that your system has a Unbound installed and it is available to the entire system (so the :command:`make install` step during installation). 
+
+The basic configuration is shown below. 
+
+.. code:: bash
+
+    server:
+        # can be uncommented if you do not need user privilige protection
+        # username: ""
+        
+        # can be uncommented if you do not need file access protection
+        # chroot: ""
+    
+        # location of the trust anchor file that enables DNSSEC. note that
+        # the location of this file can elsewhere
+        auto-trust-anchor-file: "/usr/local/etc/unbound/root.key"
+        # auto-trust-anchor-file: "/var/lib/unbound/root.key"
+    
+        # send minimal amount of information to upstream servers to enhance privacy
+        qname-minimisation: yes
+    
+        # specify the interfaces to answer queries from by ip-address.
+        interface: 0.0.0.0
+        # interface: ::0
+    
+        # addresses from the IP range that are allowed to connect to the resolver
+        access-control: 192.168.0.0/16 allow
+        # access-control: 2001:DB8/64 allow
+
+By default Unbound comes with `chroot <https://wiki.archlinux.org/title/chroot>`_ enabled. This provides an extra layer of defence against remote exploits. If :command:`chroot` gives you trouble, you can enter file paths as full pathnames starting at the root of the filesystem (``/``) or disable it in the config if this feature is not required.
+
+.. code::bash
+
+	# disable chroot
+	chroot: ""
+
+Unbound assumes that a user named "unbound" exists. You can add this user with your favourite account management tool (:command:`useradd(8)`), or disable the feature with ``username: ""`` in the config. If it is enabled, after the setup, any other user privileges are dropped and the configured username is assumed.
```

Comment:
Is `useradd` available on all systems? I seem to recall it being `adduser` instead in some cases. Perhaps best to just to leave the `useradd` reference out? What does disabling the feature mean - does Unbound then run as whichever user launched its process?